packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -81,94 +124,100 @@
             :class="[$style['c-accountInfo-changePasswordButton']]"
             data-test-id="account-info-submit-button"
             button-type="secondary"
+            href="/change-password?returnurl=/account/info"
             button-size="large"
             is-full-width
             action-type="submit">
             {{ $t('buttons.changePassword') }}
         </f-button>
 
-        <hr>
-
-        <p>{{ $t('deleteAccountMessage') }}</p>
-
-        <f-link
-            is-distinct
-            href="/help"
-            target="_blank">
-            {{ $t('deleteAccountLink') }}
-        </f-link>
-    </card>
+        <delete-account />
+    </f-card>
 </template>
 
 <script>
 import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 
-import Card from '@justeat/f-card';
+import FErrorMessage from '@justeat/f-error-message';
+import '@justeat/f-error-message/dist/f-error-message.css';
```

Comment:
I think there was a discussion in the #frontendguid channel to keep these css import in the style section.